kv/dist_sender.go

```diff
@@ -496,11 +503,17 @@ func (ds *DistSender) Send(call client.Call) {
 		args.Header().Timestamp = ds.clock.Now()
 	}
 
+	// If this is an InternalPushTxn, set ignoreIntents option as necessary.
```

Comment:
`+ // This prevents a potential infinite loop; see the comments in proto.InternalLookupRangeRequest.`